src/cleanvision/dataset/hf_dataset.py

```diff
@@ -27,8 +27,13 @@ def __init__(self, hf_dataset: "datasets.Dataset", image_key: str):
     def __len__(self) -> int:
         return len(self._data)
 
-    def __getitem__(self, item: Union[int, str]) -> Image.Image:
-        return self._data[item][self._image_key]
+    def __getitem__(self, item: Union[int, str]) -> Optional[Image.Image]:
+        try:
+            image = self._data[item][self._image_key]
+            return image
+        except Exception as e:
+            print(f"Could not load image at index: {item}", e)
+            return None
```

Comment:
would be good if you can update the PR description to clarify at the Imagelab level what the user experience is like when there is an image like this in the dataset.

Does the entire call to find_issues fail or does it just skip over this image?
The latter behavior would be desirable in future if it's not implemented, but need to think about how to score such an image / other details

Reply:
Added details about the change in description